app/move/views/view.njk

```diff
@@ -60,12 +60,50 @@
   </header>
 {% endblock %}
 
+{% set urls = {
+  personal_details: '/personal-details',
```

Comment:
This feels like logic that shouldn't live in a view layer. Maybe a middleware/presenter could be used to build links for the sections and use the key as reference in the view.

Reply:
I left it so for discussion. Ideally we could introspect this info from the steps themselves but we don't have a key apart from the final part of the url itself. Should we add a `key` property to the steps or am I missing something?

Reply:
Now handled in `view` controller